executor/index_lookup_merge_join.go

```diff
@@ -352,7 +351,7 @@ func (omw *outerMergeWorker) buildTask(ctx context.Context) (*lookUpMergeJoinTas
 		results:     make(chan *indexMergeJoinResult, numResChkHold),
 		outerResult: chunk.NewList(omw.rowTypes, omw.executor.base().initCap, omw.executor.base().maxChunkSize),
 	}
-	task.memTracker = memory.NewTracker(stringutil.MemoizeStr(func() string { return fmt.Sprintf("lookup join task %p", task) }), -1)
+	task.memTracker = memory.NewTracker(0, -1)
```

Comment:
What's the difference between `0` and `-1`?

Reply:
ditto. The original tracker label is also useless? So I just use 0 as the label here.

`-1` was the original value.

Reply:
Already use the `LabelForSimpleTask` here.